java/jsf/kubernetes/Application/charts/service-a/templates/_helpers.tpl

```diff
@@ -1,32 +0,0 @@
-{{/* vim: set filetype=mustache: */}}
```

Comment:
Why have you deleted these files? Should the commit show rename here as well?

Reply:
I didn't delete the file actually. I will double check on this